buildtools/build.sh

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env bash
 
-set -e
+set -eEu -o pipefail
 
 PROJECT=""
+WEBPACK_MODE="--mode production"
+MODE="${1:-""}"
```

Comment:
Just `MODE="${1:-}"` maybe? Feels a bit more idiomatic.

Reply:
You're right, done, thanks :)